src/questionnaireitem_defined_selector.js

```diff
@@ -45,8 +45,13 @@ QuestionnaireItemDefinedSelector.prototype._createAnswerNode = function() {
     return node;
 };
 QuestionnaireItemDefinedSelector.prototype._handleChange = function(event) {
-    this.answer = this.select.value;
-    this._sendReadyStateChanged();
+    if (this.select.value != "") {
```

Comment:
Please add a comment, why this is necessary.

Reply:
Added a comment. This problem seems to be browser independent. Occurs in Chrome & Firefox for me.